pandas/core/arrays/base.py

```diff
@@ -610,3 +618,121 @@ def _ndarray_values(self):
         used for interacting with our indexers.
         """
         return np.array(self)
+
+
+class ExtensionOpsBase(object):
+    """
+    A base class for the mixins for different operators.
+    Can also be used to define an individual method for a specific
+    operator using the class method create_method()
+    """
+    @classmethod
+    def create_method(cls, op):
```

Comment:
can you make this a private method? (`_create_method`)

Reply:
@jorisvandenbossche See my reply here https://github.com/pandas-dev/pandas/pull/21261#discussion_r192418506 as to why I think it should not be private.

Reply:
It's fine to have it private for *users* (and I think that is better, as a end-user should never have to use this), but have it documented for extension array authors. We have other methods like that on the ExtensionArray class as well, like `_from_sequence`. 
We just need to clearly document it

Reply:
@jorisvandenbossche OK, that makes sense.  I'll make that change.